pkgs/yukiarrr/ecsk/pkg.yaml

```diff
@@ -0,0 +1,3 @@
+---
+packages:
+  - yukiarrr/ecsk@v0.9.3
```

Comment:
```suggestion
  - name: yukiarrr/ecsk@v0.9.3
```

pkg.yaml is broken.
From the next time, could you use `cmdx s` according to the guide?

https://aquaproj.github.io/docs/products/aqua-registry/contributing/#use-cmdx-s-definitely